src/masternodes/operators.cpp

```diff
@@ -33,10 +33,11 @@ Res COperatorView::UpdateOperator(const COperatorId& operatorId, const COperator
         return Res::Err("operator <%s> not found", operatorId.GetHex());
     }
 
+    //move everithing one by one(if possible) for the clarity.
     opertr.operatorAddress = std::move(newOperator.operatorAddress);
-    opertr.operatorName = newOperator.operatorName;
-    opertr.operatorURL = newOperator.operatorURL;
-    opertr.operatorState = newOperator.operatorState;
+    opertr.operatorName = std::move(newOperator.operatorName);
+    opertr.operatorURL = std::move(newOperator.operatorURL);
+    opertr.operatorState = std::move(newOperator.operatorState);
```

Comment:
Let me explain a bit. It was introduced by oracle PR but moving from const object can not be done. You have `const COperator` so moving its members are unaffected. You should declare `newOperator` non const.

Reply:
Aha it's a const ref. thanks for pointing out. make sense. But i think we should keep it as const ref. otherwise, it means you can change the incoming object properties inside the function.